src/components/ApplicationInfo/index.tsx

```diff
@@ -39,10 +39,10 @@ export const ApplicationInfo: React.FC<ApplicationInfoProps> = ({
   const logoPath = useAppSelector(state => state.logoPath);
   const appDescription = useAppSelector(state => state.description);
 
-  const [isVisible, setVisible] = useState<boolean>(false);
+  const [visible, setVisible] = useState<boolean>(false);
```

Comment:
Personally I would prefer `isVisible`. Probably this can be fixed by renaming the setter to `setIsVisible`?

Reply:
I think so, and can do this

Reply:
done in dd02a0b4e3316131a38e3fffea6b5dc527b3077a